src/nfa/operations.cc

```diff
@@ -34,9 +37,14 @@ using namespace Mata::Nfa;
 using Mata::Symbol;
 using Mata::BoolVector;
 
+// Vector of bool is probably slower than vector of char (from some little experiment).
+// Just BoolArray would be better?
+// It would make sense to make this type name global, it is type that we use globally to represent bool vectors. ?
 using StateBoolArray = std::vector<bool>; ///< Bool array for states in the automaton.
 
 namespace {
+    //Remove direct_, there are no close alternatives for NFA (unlike for Buchi). Remove "compute_" ?
```

Comment:
I would keep `compute_` prefix. The idiom is to use verbs for function names, it reads more naturally, in my opinion, and it leaves no place for questions about what the function does. If you call `fw_simulation()`, I do not know whether it initializes something, returns something, has to compute something first and then return it, or whether it only checks for something. This is why `get_` (something already existing, precomputed or easily comuted), `compute_` (something more expensive which has to be computed) are useful indicators.

Reply:
Related to above comments, I propose to use nouns describing the returned thing whenever no misunderstanding is possible and use get_/construct_ etc only when a misunderstanding is possible.